inst/js/js.cookie.min.js

```diff
@@ -0,0 +1,2 @@
+/*! js-cookie v3.0.5 | MIT */
```

Comment:
Please move this to its own folder and add the respective LICENSE there. See #159 _(you can copy that `LICENSE` file from there)_ which follows the example of [`{shiny.react}`](https://github.com/Appsilon/shiny.react/tree/main/inst/www/react)